README.md

```diff
@@ -1,30 +1,4 @@
-[![CircleCI](https://circleci.com/gh/giantswarm/template.svg?style=shield)](https://circleci.com/gh/giantswarm/template)
+# workload-identity-gcp-utils
 
-# REPOSITORY_NAME
+A collection of scripts to help you configure and use workload identity on GCP.
```

Comment:
I'd add a few words saying that just running `make` would create all the service accounts required by GS.

Reply:
Done, also added `make help` entries